app/controllers/claims_form_callbacks.rb

```diff
@@ -1,50 +1,8 @@
 module ClaimsFormCallbacks
-  def current_school_before_show
-    set_backlink_override_to_current_slug if on_school_search_results?
-  end
-
-  def claim_school_before_show
-    set_backlink_override_to_current_slug if on_school_search_results?
-  end
-
   def teaching_subject_now_before_show
```

Comment:
I think this one could be handled by the additional payments slug sequence. Not a blocker for this pr, have opened a separate ticket https://dfedigital.atlassian.net/browse/CAPT-2152

Reply:
i tried this one theres an explicit test for it which would need to be "modified". there isn't really a mechanism at the moment to allow for this kinda functionality without writing explicit controllers to bounce the use to correct place.